src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -14,6 +28,31 @@
 
     @Override
     public int getSize() {
-        return 0;
+        return size;
+    }
+
+    private int getHash(K key) {
+        if (key == null) {
+            return 0;
+        }
+        return Math.abs(key.hashCode() % table.length);
+    }
+
+    private void resize() {
+        Node<K, V>[] currentTable = table;
```

Comment:
The 'resize' method is incomplete. You need to copy the existing entries to the new table with updated indexes after resizing.